src/js/components/Popover.jsx

```diff
@@ -2,22 +2,23 @@
 
 import React, { Component, PropTypes } from 'react';
 import FloatingElement, { positions, positionValues, sanitizePosition } from './FloatingElement';
+import {ModalContainer} from '.';
 
 export class Popover extends Component {
 
     //--------------------------------------
     //  User Interaction
     //--------------------------------------
 
-    modalScreenClicked:HTMLEventHandler = () => {
+    modalScreenClicked = () => {
```

Comment:
Ah, I take this is another way of binding a class method's scope so that `this` is the class instance? Probably more performant (and readable) than using fat arrow within the `render` method?